lighthouse-core/report/v2/renderer/dom.js

```diff
@@ -127,6 +127,28 @@ class DOM {
   }
 
   /**
+   * @param {string} text
+   * @return {!Element}
+   */
+  convertMarkdownCodeSnippets(text) {
```

Comment:
do you not want to fold this in to `createSpanFromMarkdown`? if not can we rename to `convertMarkdownLinkSnippets` or something?